packages/esl-website/views/_layouts/content.njk

```diff
@@ -7,7 +7,7 @@
 
   <main class="content {{ containerCls or 'container' }} {{ 'container-aside' if aside }} {{ 'markdown-container' if isMarkdownSource }}">
     {% set isMarkdownSource = page.templateSyntax.includes('md') %}
-    {% if title %}
+    {% if title and not landing %}
```

Comment:
Probably we can replace the `title` with the `name` instead of introducing a new marker? (in case the original reson is having page name in navigation)